src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 #spring.datasource.driver-class-name=org.postgresql.Driver
-#spring.datasource.url=jdbc:postgresql://172.23.0.2:5432/upperio
-#spring.datasource.username=upperio_user
-##spring.datasource.password=upperio//s3cr37
+#spring.datasource.url=jdbc:postgresql://localhost:5432/test_db
+#spring.datasource.username=postgres
+#spring.datasource.password=postgres
```

Comment:
Is this something need to be reverted ?

Reply:
its commented out anyway, but i think generic is always better